.github/main.workflow

```diff
@@ -0,0 +1,11 @@
+workflow "on push" {
+  on = "push"
+  resolves = ["GitHub Action for pylint"]
+}
+
+action "GitHub Action for pylint" {
+  uses = "PyCQA/pylint/github_actions@master"
+  # args = "pip install -r requirements.txt ; pylint **/*.py"
+  args = "pip install . ; pylint **/*.py"
```

Comment:
IMO it would be best to make `args = **/*.py` and not allow arbitrary shell commands. The best practice with GH Actions would be to use special purpose steps if any setup is needed.

For reference: https://github.com/ScottBrenner/cfn-lint-action